drafts/pair-programming-mistakes.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Navigating the Challenges of Pair Programming
+subtitle: Lessons from My Journey
+domain: software-engineering-corner.zuehlke.com
+tags: pair-programming, software-development
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1717162463753/6mZkSMtlz.png?auto=format
+publishAs: oliveramstutz
+hideFromHashnodeCommunity: false
+saveAsDraft: true
+---
+
```

Comment:
is now grouped 👍

Reply:
in general, all sentences that are split by one new line is in the end grouped together into a section.
Also imo it makes it easier to review the text, since you can easily write a comment for one sentence instead of having to write one for a whole bunch of sentences.
both is fine, just an fyi ;)